projects/step-frontend/src/lib/modules/execution/components/schedule-overview/schedule-overview.component.ts

```diff
@@ -400,4 +420,49 @@ export class ScheduleOverviewComponent implements OnInit {
     let now = DateTime.now();
     this.updateRange({ start: now.minus({ millisecond: time }), end: now });
   }
+
+  private createErrorsChart(taskId: string, timeRange: TimeRange) {
+    let groupingAttribute = this.errorChartSelectedGrouping.attribute;
+    const request: FetchBucketsRequest = {
+      start: timeRange.from,
+      end: timeRange.to,
+      numberOfBuckets: 1,
+      oqlFilter: `attributes.taskId = ${taskId}`,
+      groupDimensions: [groupingAttribute],
+      collectAttributeKeys: ['status'],
+      collectAttributesValuesLimit: 10,
+    };
+    this._timeSeriesService.getReportNodesTimeSeries(request).subscribe((response) => {
+      let totalCount = 0;
+      let countWithoutError = 0;
+      const data: TableErrorEntry[] = response.matrixKeys
+        .map((key, index) => {
+          const error = key[groupingAttribute];
+          const bucket = response.matrix[index][0];
+          const bucketCount = bucket.count;
```

Comment:
Just general feedback, but I think this const doesn't give any benefit to bucket.count

Reply:
@neogucky I use it in 3 places, only that's why I extracted. both options are fine for me